extensions.json

```
@@ -72,6 +72,11 @@
       "repository": "https://github.com/appulate/vscode-file-watcher",
       "version": "1.0.3"
     },
+    {
+      "id": "asvetliakov.vscode-neovim",
+      "repository": "https://github.com/asvetliakov/vscode-neovim",
+      "version": "0.0.50"
```

Since this project has dedicated release tags, it's probably better to release version `0.0.50` from the tag `v0.0.50` rather than from the default branch.

```suggestion
      "version": "0.0.50",
      "checkout": "v0.0.50"
```